Application/Setting/User/Account/Service/Entity/TblUserAccount.php

```diff
@@ -240,4 +262,75 @@ public function setGroupByCount($GroupByCount)
     {
         $this->GroupByCount = $GroupByCount;
     }
+
+    /**
+     * @return string
+     */
+    public function getAccountCreator()
+    {
+        return $this->AccountCreator;
+    }
+
+    /**
+     * @param string $AccountCreator
+     */
+    public function setAccountCreator($AccountCreator = '')
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$AccountCreator" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)